asaps/models.py

```diff
@@ -42,6 +42,13 @@ def save_record(self, rec_obj):
         logger.info(response.json())
         return response.json()
 
+    def get_aos_for_resource(self, uri, aolist):
```

Comment:
I think that instead of passing in an external archival object list that gets appended to, you might want to return the list of archival objects associated with this resource, and determine elsewhere what you want to do with that list. Maybe you're appending it to an existing list of archival objects, maybe you want to do something else with it. Returning just the list for this resource will make this method more reusable for different purposes.

Reply:
That is a fantastic point, I will change it